src/nvidia_rag/ingestor_server/nvingest.py

```
@@ -81,26 +92,26 @@ def get_nv_ingest_ingestor(
-        "extract_audio_params": {"segment_audio": CONFIG.nv_ingest.segment_audio},
-        "extract_page_as_image": CONFIG.nv_ingest.extract_page_as_image,
+        "extract_audio_params": {"segment_audio": config.nv_ingest.segment_audio},
+        "extract_page_as_image": config.nv_ingest.extract_page_as_image,
     }
-    if remove_extract_method or CONFIG.nv_ingest.pdf_extract_method in ["None", "none"]:
+    if remove_extract_method or config.nv_ingest.pdf_extract_method in ["None", "none"]:
         extract_kwargs.pop("extract_method")
     else:
         logger.info(
-            f"Extract method used for ingestion: {CONFIG.nv_ingest.pdf_extract_method}"
+            f"Extract method used for ingestion: {config.nv_ingest.pdf_extract_method}"
```

Ruff (0.14.5)

Logging statement uses f-string

(G004)